### PR TITLE
Add sentiment score to comment and display as emoji face

### PR DESCRIPTION
**Pull request by ALai2**
A sentiment score is requested for the comment when it is created sent by the post request to the server. The score is saved inside the comment class and returned in the json to script.js where an if else statement uses a range of values to determine which emoji to show based on the score.

### CODE REVIEW

**Comment by ALai2** [comment]
Question: Is there a specific error to handle for the language api similar to how there is TranslateException for the translation api?

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -23,12 +23,14 @@
   private String name;
   private String comment;
   private long timestamp;
+  private double score;
 
-  public Comment(long id, String name, String comment, long timestamp) {
+  public Comment(long id, String name, String comment, long timestamp, double score) {
```
Comment (linxiz) [inline portfolio/src/main/java/com/google/sps/data/Comment.java:36]: nit: you might want to add comment or javadoc to describe each param, e.g. what specific score it refers to? what does it mean when the score is high and when it is low? what is the valid range of the score?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -86,15 +92,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
+      DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+      datastore.put(commentEntity);
 
-    // Redirect back to the Comment page.
-    response.sendRedirect("/comments.html");
+      // Redirect back to the Comment page.
+      response.sendRedirect("/comments.html");
+
+    } catch (Exception error) {
+      String errorMessage = "Google NLP API returned an error " + error.getMessage();
```
Comment (linxiz) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: update the error message to display to user without the detailed impl details

**Review by linxiz — APPROVED** [review]
nit: could you also resolve the previous comments?

**Pull request merged**